sagemaker-python-sdk/pytorch_batch_inference/docker/config.properties

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+inference_address=http://0.0.0.0:8080
+management_address=http://0.0.0.0:8081
+number_of_netty_threads=32
+job_queue_size=1000
+model_store=/opt/ml/model
+#decode_input_request=false
+#load_models= flores_modified.mar
```

Comment:
@nskool please remove the commented lines

Reply:
Done, thanks